src/components/ResultsTable.tsx

```diff
@@ -18,27 +18,156 @@ export function ResultsTableWidget (props: ResultsTableProps) {
+                                            
+                                            
+                                            <td>
+                                                <table>
+                                                <tr>
+                                                    
+                                                    <th className={'nested_table'}>Name on Title </th>
+                                                   
+                                                    
+                                                    <th className={'nested_table'}>Contacts</th>
+                                                    
+                                                   
+                                                </tr>
+                                                
+                                                {nameOnTitleProperty_list.map((property_details, index_property) => (
+                                                    <tr data-index={index_property}> 
+                                                        <td>{property_details.Name_On_Title.name || 'Name on Title is not found'} &nbsp;
+                                                             <br></br>
+                                                            {/* {JSON.stringify(nameOnTitleProperty_list)} */}
+                                                        </td>  
+                                                        
+                                                        <td>
+                                                        
+                                                        {nameOnTitleContact_list.map((contact_details, index_contact) => {
+                                                            if (nameOnTitleContact_list.length > 0){
+                                                                console.log(JSON.stringify(nameOnTitleContact_list).includes(contact_details.Name_On_Title.id));
+                                                                // let list_str = "".join(str(elem) for elem in nameOnTitleContact_list)
+                                                                if (JSON.stringify(nameOnTitleContact_list).includes(contact_details.Name_On_Title.id) === false){
+                                                                    return (
+                                                                        <span>{'Contact is not found'} <br></br></span> 
+                                                                    )
+                                                                }
+                                                                else{
+                                                                    if (contact_details.Name_On_Title.id.concat(contact_details.Name_On_Title.name)  === property_details.Name_On_Title.id.concat(property_details.Name_On_Title.name)){
```

Comment:
I don't think this would ever be true. 
![CleanShot 2022-08-31 at 15 54 19](https://user-images.githubusercontent.com/1034890/187604165-6d8ef4d2-2bb6-4ce9-af02-018fdccc8167.png)

Reply:
fixed